lib/wda/webdriveragent.js

```diff
@@ -374,9 +393,24 @@ class WebDriverAgent {
     this.webDriverAgentUrl = this.url.href;
   }
 
-  async quitAndUninstall () {
+  /**
+   * Quit and uninstall running WDA.
+   * Uninstall given updatedWDABundleId, 'WDA_RUNNER_BUNDLE_ID' or running 'productBundleIdentifier
+   * for Xcode 11. Uninstall 'WDA_BUNDLE_ID' for Xcode 10-.
+   *
+   * @param {string} updatedWDABundleId The bundle id which will be uninstalled
+   */
+  async quitAndUninstall (updatedWDABundleId = WDA_RUNNER_BUNDLE_ID) {
+    const status = await this.getStatus();
```

Comment:
Can we fallback to read the id from the actual plist bundle since `status` might not be available at this stage

Reply:
Sounds good.
Here is only called by 'useNewWDA' is true